source/_integrations/generic_thermostat.markdown

```diff
@@ -85,12 +85,14 @@ precision:
   default: "`0.5` for Celsius and `1.0` for Fahrenheit."
 {% endconfiguration %}
 
-A full configuration example looks like the one below. `min_cycle_duration` and `keep_alive` must contain at least one of the following entries: `days:`, `hours:`, `minutes:`, `seconds:` or `milliseconds:`.
+__Notes__  
```

Comment:
Let's remove this, there is no need to create an emphasis here.

Reply:
no problem, just wanted to make it as a separate paragraph

Reply:
That is not how paragraphs work ;)
But in all honesty, a simple note doesn't need a paragraph. Especially since this note concerns the configuration that is listed above.